book/src/keymap.md

```diff
@@ -218,6 +218,8 @@ Jumps to various locations.
 | `n`   | Go to next buffer                                | `goto_next_buffer`         |
 | `p`   | Go to previous buffer                            | `goto_previous_buffer`     |
 | `.`   | Go to last modification in current file          | `goto_last_modification`   |
+| `w`   | Word-wise jump mode                              | `jump_to_identifier_label` |
+| `/`   | Character search jump mode                       | `jump_to_char_label`       |
```

Comment:
```suggestion
| `/`   | Character search jump mode                       | `jump_to_str_label`       |
```

According to `helix-term/src/commands.rs`, line 480.

Reply:
Oh thanks a lot for catching this out! Fixed :)